config/test/test.config.js

```diff
@@ -12,15 +12,27 @@ module.exports = {
     },
     restoreMocks: true,
     resetMocks: true,
-    testEnvironment: "jest-environment-node",
-    testMatch: ["<rootDir>/**/*.test.js"],
+    testEnvironment: "jsdom",
+    testMatch: [
+        "<rootDir>/**/*.test.js",
+        "<rootDir>/**/*_test.js",
+        "<rootDir>/**/*_test.jsx",
+    ],
+    timers: "fake",
```

Comment:
Given this, do we need to do the `jest.useFakeTimers()` in `config/test/test-setup.js`?